helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -103,6 +105,21 @@ resource/logs:
     {{- end }}
 {{- end }}
 
+{{/*
+Include logs processor
+*/}}
+{{- define "splunk-otel-collector.includeLogsProcessors" -}}
+# If .Values.logsCollection.useSplunkIncludeAnnotation is set to true, only logs for pods with splunk.com/include annotation are ingested
```

Comment:
`.Values.logsCollection.useSplunkIncludeAnnotation` is wrong I believe. But do we need to mention the condition in the unconditional helper? Maybe just remove this line? Seems like there are enough other comments added already